### PR TITLE
#400 update go.mod

### PR DESCRIPTION
**Pull request by sitikanthasezzle**
Update jwt-go library from dgrijalva/jwt-go to form3tech-oss/jwt-go
#400

### CODE REVIEW

**Comment by benjivesterby** [comment]
The correct import is https://github.com/golang-jwt/jwt according to the README this is where maintainership was migrated to.

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/fatih/structs v1.1.0
-	github.com/golang-jwt/jwt v3.2.1+incompatible
+	github.com/form3tech-oss/jwt-go v3.2.1+incompatible
```
Comment (benjivesterby) [inline go.mod:7]: This repository is archived. According to the readme from `dgrijalva/jwt-go` the correct maintenance of this repository was moved to `github.com/golang-jwt/jwt` so the current import is correct.

**Review by benjivesterby — CHANGES_REQUESTED** [review]
address comment

File: go.mod (hunk 1 above)
Comment (sitikanthasezzle) [reply go.mod:7]: github.com/andygrunwald/go-jira@v1.13.0 requires\n\tgithub.com/dgrijalva/jwt-go@v3.2.0+incompatible: missing go.sum entry; to add it:\n\tgo mod download github.com/dgrijalva/jwt-go\n

**Comment by benjivesterby** [comment]
Based on comments in the dependabot PR this needs to be upgraded to V4. I've put in a PR for the upgrade #403

File: go.mod (hunk 1 above)
Comment (sitikanthasezzle) [reply go.mod:7]: I am getting this error @benjivesterby

**Pull request closed without merging**